spec/jsSpecs/ares-log.spec.js

```diff
@@ -123,10 +132,30 @@ describe(aresCmd + " -n 2", function() {
                 if (options.device === "emulator") {
                     expect(stderr).toContain("ares-log ERR! [Tips]: Unable to connect to the target device. root access required");
                 }
-            } else {
+            } else if (targetLogDaemon === "journald") {
                 expect(stdout).not.toContain("[Info] Set target device : " + options.device);
                 expect(stdout).toContain("-- Journal begins at");
-                expect(stdout.match(logRegExp).length).toBe(2);
+                expect(stdout.match(journalLogRegExp).length).toBe(2);
```

Comment:
What is toBe(2)? is It length value?

Reply:
The goal of this test is to output 2 lines of log. Compare the length to check whether 2 log lines that satisfy the regular expression are normally output.

Reply:
okay!